execution of 'R: Update github.io' with '10.3.3' faililng

https://github.com/checkstyle/checkstyle/runs/8170868585?check_suite_focus=true

```
Run ./.ci/update-github-io.sh 10.3.3
TARGET_VERSION=10.3.3
error: pathspec 'checkstyle-10.3.3' did not match any file(s) known to git
Error: Process completed with exit code 1.
```

please do `git fetch ---tags` after clone.

--- a/.github/workflows/update_github_io.yml
+++ b/.github/workflows/update_github_io.yml
@@ -15,8 +15,9 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout the latest code
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
         with:
+          fetch-depth: 0
           token: ${{ secrets.GITHUB_TOKEN }}
       - name: Run Shell Script
         run: |

